### ISSUE TITLE
Schlagworte Standartisieren

### ISSUE THREAD

**Issue by johappel** [rpi-virtuell/rw-materialpool#333]
Schlagworte sollten nie im Plural sein

Problembeispiel:
![grafik](https://user-images.githubusercontent.com/307023/29263786-aa405b16-80da-11e7-84e3-bcec1264b1cb.png)

**Comment by staude** [comment]
Anmerkung von mir aus dem Chat

gerade bei der vorbereitung eines vortrags zum thema suche usw über diese schlagworte gestolpert
https://material.rpi-virtuell.de/wp-admin/edit-tags.php?taxonomy=schlagwort&post_type=post&s=jesus

jesus ist doch ein synonym von jesus christus ( https://material.rpi-virtuell.de/wp-admin/edit.php?s=jesus&post_status=all&post_type=synonym&action=-1&m=0&paged=1&action2=-1 ) , sollte also eigentlich nicht als eigenes schalgwort existieren. und "jesus und die kinder" als Schalgwort? sollte das nicht lieber einmal schlagwort kinder und einmal jesus sein?

**Comment by retinasilk** [comment]
Ja, für Jesus - siehe: http://d-nb.info/gnd/118557513 - "Jesus und die Kinder" ist meilenweit von einem Schlagwortcharakter entfernt. Das ist bestenfalls ein Titel, weil dieses Literal als Titel in der Bibel vorkommt. Vorzugsbenennung: "Jesus Christus" 

Im Übrigen gibt die Schlagwortnormdatei und deren Vorzugsbenennung. Begriffe immer im Singular, es sei denn, dass der Begriff auch in der Normdatei im Plural steht.

**Comment by retinasilk** [comment]
Valides Plural/Singular-Beispiel: Salmonella / Synonym: Salmonellen.

**Comment by retinasilk** [comment]
Reines Pluralbeispiel: Eltern

**Comment by lehr-ruetsche** [comment]
Ich habe mit Kerstin Höbing einen Termin ausgemacht während der nächsten GKWM im CI. Da möchte ich alle diese Probleme ansprechen, denn manchmal ist echt die Einzahl und die Mehrzahl sinnvoll. Es gibt auch noch andere Probleme. Sie verwenden zum Beispiel auch nicht immer Jesus Christus, weil das nicht immer sinnvoll ist, sondern auch Jesus von Nazareth.

**Comment by johappel** [comment]
@lehr-ruetsche Das ist sehr gut. Aber wir sollten m.E. nicht Singular und Plural für das gleiche Sachwort (Bild, Bilder)  verwenden: einmal würde links in der Facette sowohl Bild als auch Bilder auftauchen und beide Begriffe wahrscheinlich noch unterschiedliche Ergebnisse ausliefern.

**Comment by lehr-ruetsche** [comment]
Klärendes Gespräch mit der Dok-Abteilung des CI wurde geführt. Bleibe in Kontakt. Kann geschlossen werden.
